ge0/url_generator.cpp

```diff
@@ -103,6 +103,24 @@ std::string GenerateShortShowMapUrl(double lat, double lon, double zoom, std::st
   return urlSample;
 }
 
+std::string GenerateCoordUrl(double lat, double lon, double zoom, std::string const & name)
+{
+  static const std::string kHttpSharePrefix = "https://omaps.app/";
+  std::string url = kHttpSharePrefix;
```

Comment:
No need in introducing unnecessary allocation and copying here:
```suggestion
  std::string url = "https://omaps.app/";
```